camelot/core/serializable.py

```diff
@@ -69,6 +74,16 @@ def default(self, obj):
             return obj.toVariant()
         if isinstance(obj, QtGui.QColor):
             return obj.name()
+        if isinstance(obj, (Decimal, datetime.date, datetime.datetime)):
+            return initial_naming_context._bind_object(obj)
+        if isinstance(obj, StoredFile):
+            return obj.verbose_name # FIXME: not sure if this is enough...
+        if isinstance(obj, QtGui.QImage):
```

Comment:
weet nt of dat hier een goed idee is, kunnen we mss beter in de delegate doen, en later eventueel nog hoger verhuizen, of in een gespecialiseerde context steken, zodat dit kan gecached w

Reply:
Cachen is mss geen slecht idee. We kunnen dan een sha1 ofzo gebruiken en enkel de image data ophalen indien de client de hash niet kent ofzo?